routers/users.py

```
@@ -181,3 +184,33 @@ async def get_user_profile(user_id: int):
+        "user_id": user["id"]
+    }
+    new_access_token = auth.create_access_token(access_token_data)
+    new_refresh_token = auth.create_refresh_token(user["id"])
+
+    return Token(access_token=new_access_token, refresh_token=new_refresh_token, token_type="bearer")
+
+
+@router.post("/logout", status_code=status.HTTP_204_NO_CONTENT)
+async def logout(
+        token_request: RefreshTokenRequest,
+        current_user: dict = Depends(auth.get_current_user)
```

Ruff (0.14.11)

Unused function argument: `current_user`

(ARG001)